### ISSUE TITLE
Option to Break Tabs to New Windows and Combine Windows back to Tabs

### ISSUE THREAD

**Issue by asjones987** [nomacs/nomacs#261]
Maybe I am missing this, but I would like a way so that if I have 5 tabs open I can drag some or all of them out to form new Windows/Instances.  May have one window with 2 tabs and one window with 3.... etc

At the same time if I have images in separate Windows/instances I can drag/move them back to the same Window and just have a bunch of tabs.

thanks

Alan

**Comment by diemmarkus** [comment]
thanks for the suggestion. I guess you mean a similar behavior like chrome/firefox?!

**Comment by asjones987** [comment]
Yes similar to Firefox and Chrome. In many ways I see this feature as something related to #95  & #260  if an image is set to open in an new tab I will want to "break" that to new windows an if I set it the other way I will want to combine windows to tabs :)

I know the work is very different from the other, but use wise they may be related.

thanks
